TestCntlrStub/src/ts_utls.c

```diff
@@ -1765,11 +1765,11 @@ int tsSendResetReq(unsigned int resetType, int numOfUes, unsigned char *ueLst)
       ((ResetReq*)msgptr)->cause.causeVal = 1;
 
       ((ResetReq*)msgptr)->r.partialRst.numOfConn = numOfUes;
-      ((ResetReq*)msgptr)->r.partialRst.ueIdLst = malloc(numOfUes);
-      memcpy(((ResetReq*)msgptr)->r.partialRst.ueIdLst, ueLst, numOfUes);
+      ((ResetReq*)msgptr)->r.partialRst.ueS1apIdPairList = malloc(numOfUes);
```

Comment:
This malloc memory typecasting looks invalid. "ueIdLst" was a list of U32 UEId values, whereas "ueS1apIdPairList" is a list with structures of 3 U32 values inside each. I believe, this should be like:
((ResetReq*)msgptr)->r.partialRst.ueS1apIdPairList = 
   (NbUeS1apIdPair) malloc(sizeof(NbUeS1apIdPair) * numOfUes);